public/plugin.tsx

```diff
@@ -157,8 +204,21 @@ export class SecurityPlugin implements Plugin<SecurityPluginSetup, SecurityPlugi
     if (config.multitenancy.enabled) {
       addTenantToShareURL(core);
     }
+
+    if (config.multitenancy.enable_aggregation_view) {
+      const columns = deps.savedObjectsManagement.columns.getAll();
+    }
+
     return {};
   }
 
   public stop() {}
 }
+
+function isPrivateTenant(selectedTenant: string) {
```

Comment:
Thinking about code organization, what do you think about moving this and `isGlobalTenant` to a util file?

Reply:
done